### PR TITLE
Fixes #7050: Add a rudder server disable command

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/7050

### CODE REVIEW

**Comment by jooooooon** [comment]
Commit modified

File: share/commands/server-disable-policy-distribution [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+# @description Stop Rudder from distributing new policies as a server
+# @man This is useful when you want to temporarily prevent your Rudder server
+# @man from doing any changes on your agents
+
+# First, prevent the agent from restarting cf-serverd
+touch /opt/rudder/etc/disable-policy-distribution
```
Comment (jooooooon) [inline share/commands/server-disable-policy-distribution]: This depends on rudder-techniques#929

**Comment by jooooooon** [comment]
Normation/rudder-techniques#929 has been merged now. @peckpeck can you review this please?

File: share/commands/server-disable-policy-distribution [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+# Then, actually stop the server right now
+if type killall > /dev/null 2> /dev/null; then
+  killall -1 cf-serverd
```
Comment (peckpeck) [inline share/commands/server-disable-policy-distribution]: 1 is SIGHUP, this is made for reloading not stopping

**Comment by jooooooon** [comment]
Commit modified

File: share/commands/server-disable-policy-distribution (hunk 2 above)
Comment (jooooooon) [inline share/commands/server-disable-policy-distribution]: Good catch, this was a lazy copy paste

**Pull request merged**